src/Util/ParatestFunction.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ParaTest\Util;
+
+use function array_splice;
+
+final class ParatestFunction
+{
+    /**
+     * @param array<int, string> $arg
+     */
+    public static function createScriptForParatest(array &$arg, string $dir): string
+    {
+        unset($arg[1]);
+        array_splice($arg, -3);
+        $script = $dir . '/paratest';
+        $arg[]  = '--log-teamcity';
+        $arg[]  = 'php://stdout';
+
+        return $script;
+    }
```

Comment:
As far as I can tell, the script path has no need to be tested, so we can use this function just for the `argv` manipulation:
```suggestion
    public static function handleArgvFromPhpstorm(array &$arg, string $dir): void
    {
        unset($arg[1]);
        array_splice($arg, -3);
        $arg[]  = '--log-teamcity';
        $arg[]  = 'php://stdout';
    }
```